### ISSUE TITLE
项目引用了electron等768个开源组件，存在25个漏洞，建议升级

### ISSUE THREAD

**Issue by AbbyKissK** [gecko655/proofreading-tool#30]
大佬，看你这个项目调用了electron等768个开源组件，存在25个安全漏洞，建议你升级下。
```
漏洞标题：Google Chrome堆缓冲区溢出漏洞
漏洞编号：CVE-2021-37981
漏洞描述：
Chrome是由Google开发的一款Web浏览工具。
											
												Google Chrome 95.0.4638.54之前版本中的Skia存在堆缓冲区溢出漏洞。远程攻击者可利用该漏洞能通过精心制作的HTML页面执行沙箱逃逸。
国家漏洞库信息：https://www.cnvd.org.cn/flaw/show/CNVD-2021-84815
影响范围：[12.0.0, 12.2.3)
最小修复版本：12.2.3
引入路径：
proofreading-tool@1.0.9->electron@12.1.0
```
另外24个漏洞 ，信息有点多我就不贴了，你自己看下完整报告：https://www.mfsec.cn/jr?p=afbb99

**Comment by gecko655** [comment]
DeepL:

Big brother, look at your project calls the electron and other 768 open source components, there are 25 security vulnerabilities, we recommend that you upgrade the next.
```
Vulnerability title: Google Chrome heap buffer overflow vulnerability
Vulnerability number: CVE-2021-37981
Vulnerability description.
Chrome is a Web browsing tool developed by Google.
											
												A heap buffer overflow vulnerability exists in Skia in versions prior to Google Chrome 95.0.4638.54. A remote attacker can exploit this vulnerability to be able to perform a sandbox escape via a crafted HTML page.
National Vulnerability Database information: https://www.cnvd.org.cn/flaw/show/CNVD-2021-84815
Impact Scope: [12.0.0, 12.2.3)
Minimum Fix Version: 12.2.3
Introduction path.
proofreading-tool@1.0.9->electron@12.1.0
```
Another 24 vulnerabilities , information is a bit too much I will not post, you read the full report yourself: https://www.mfsec.cn/jr?p=afbb99

Translated with www.DeepL.com/Translator (free version)

**Comment by gecko655** [comment]
（"大佬" は、 "Hey guys" くらいの意味っぽい）
https://zawazawa.jp/share-language-community/topic/44
